README.md

```diff
@@ -73,7 +73,7 @@ to property ```destroySession```  in ```LinkedInUserWidget``` or ```LinkedInAuth
   int expiresIn;
   String profilePicture;
   String email;
-  String userId; (from version 0.1.)
```

Comment:
please leave it as there can still be people who are running this version, you can change it to 0.1.x maybe